src/taipy/gui_core/viselements.json

```diff
@@ -1,254 +1,256 @@
+              [
+                "state",
+                "State^"
+              ],
+              [
+                "var_name",
+                "str"
+              ],
+              [
+                "value",
+                "optional[Scenario]"
+              ]
+            ]
+          },
+          {
+            "name": "height",
+            "type": "str",
+            "default_value": "50vh",
+            "doc": "The maximum height, in CSS units, of the tree."
+          }
+        ]
+      }
+    ],
+    [
+      "scenario",
+      {
+        "inherits": [
+          "core_gui_shared"
         ],
-        [
-            "scenario_dag",
-            {
-                "inherits": [
-                    "core_gui_shared"
-                ],
-                "properties": [
-                    {
-                        "name": "scenario",
-                        "type": "dynamic(Scenario|list[Scenario])",
-                        "doc": "The scenario to display."
-                    },
-                    {
-                        "name": "render",
-                        "type": "dynamic(bool)",
-                        "default_value": "True",
-                        "doc": "If True, this scenario DAG is visible on the page."
-                    },
-                    {
-                        "name": "show_toolbar",
-                        "type": "bool",
-                        "default_value": "True",
-                        "doc": "If True, the DAG toolbar is visible.<br/>If False, it is not shown."
-                    },
-                    {
-                        "name": "height",
-                        "type": "str",
-                        "default_value": "50vh",
-                        "doc": "The maximum height (in CSS unit) of the DAG."
-                    },
-                    {
-                        "name": "width",
-                        "type": "str",
-                        "default_value": "50vw",
-                        "doc": "The maximum width (in CSS unit) of the DAG."
-                    }
-                ]
-            }
+        "properties": [
+          {
+            "name": "scenario",
+            "default_property": true,
+            "type": "dynamic(Scenario|list[Scenario])",
+            "doc": "The scenario to display and edit.<br/>If the value is a list, it must have a single element otherwise nothing is shown."
+          },
+          {
+            "name": "active",
+            "type": "dynamic(bool)",
+            "default_value": "True",
+            "doc": "Indicates if this component is active.<br/>An inactive component allows no user interaction."
+          },
+          {
+            "name": "expandable",
+            "type": "bool",
+            "default_value": "True",
+            "doc": "If True, the scenario visualizer can be expanded.<br/>If False, the scenario visualizer is collapsed and only its name a <i>submit</i> button are visible."
```

Comment:
I False, ... is always collapse ?
its name a submit => its name and a submit

Reply:
On it!